src/main/java/de/retest/recheck/cli/subcommands/Commit.java

```diff
@@ -78,6 +86,24 @@ public void run() {
 		}
 	}
 
+	private void inputDataInformation( final ReviewResult reviewResult ) throws IOException {
+		final InputStreamReader inputStreamReader = new InputStreamReader( System.in );
+		final BufferedReader bufferedReader = new BufferedReader( inputStreamReader );
+		final String input = bufferedReader.readLine();
+		if ( input.toLowerCase().equals( "yes" ) ) {
```

Comment:
I would suggest to split these concerns: One method that handles user input, one method that applies changes.

Maybe you also want to allow inputs such as "y" and "n"?